src/WebJobs.Extensions.DurableTask/DurableOrchestrationContext.cs

```
@@ -234,9 +234,22 @@ public Task CallFunctionAsync(string functionName, params object[] parameters)
 
             // TODO: Support for versioning
             string version = DefaultVersion;
-            this.config.AssertActivityExists(functionName, version);
+            FunctionType functionType = this.config.GetFunctionType(functionName, version);
 
-            Task<TResult> callTask = this.innerContext.ScheduleTask<TResult>(functionName, version, parameters);
+            Task<TResult> callTask;
+
+            switch (functionType)
+            {
+                case FunctionType.Activity:
+                    callTask = this.innerContext.ScheduleTask<TResult>(functionName, version, parameters);
+                    break;
+                case FunctionType.Orchestrator:
+                    callTask = this.innerContext.CreateSubOrchestrationInstance<TResult>(functionName, version, JsonConvert.SerializeObject(parameters));
+                    break;
+                default:
+                    callTask = null;
```

>callTask = null; [](start = 20, length = 16)

This will result in a null-ref below. Please throw an exception instead.